src/providers/palm/api.ts

```diff
@@ -0,0 +1,23 @@
+import { ProviderAPIConfig } from "../types";
+
+export const PalmApiConfig: ProviderAPIConfig = {
+    baseURL: "https://generativelanguage.googleapis.com/v1beta3",
+    headers: () => {
+        return { "Content-Type": "application/json" }
+    },
+    getEndpoint: (fn: string, API_KEY: string, model: string) => {
```

Comment:
API key is not required here to determine endpoint. Please remove it and also modify wherever this is called.

Reply:
API key is a query param here and is part of the URL

Reply:
Ohh. Got it.